pkg/scheduler/flavorassigner/flavorassigner.go

```diff
@@ -237,7 +237,24 @@ func lastAssignmentOutdated(wl *workload.Info, cq *cache.ClusterQueue) bool {
 // FlavorAssignmentMode.
 func AssignFlavors(log logr.Logger, wl *workload.Info, resourceFlavors map[kueue.ResourceFlavorReference]*kueue.ResourceFlavor, cq *cache.ClusterQueue, counts []int32) Assignment {
```

Comment:
I don't think we should keep this change. Even though the log lines will be disabled in production, there might still be some cost associated with formatting.

Maybe if you put all the log lines inside a single `if log.V(6).Enabled()`.

Reply:
Done, thanks